.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,29 @@
-<!--
-Thanks for submitting a pull request to ESLint. Before continuing, please be sure you've read over our guidelines:
-http://eslint.org/docs/developer-guide/contributing/pull-requests
+**What is the purpose of this pull request? (put an "X" next to item)**
 
-Specifically, all pull requests containing code require an **accepted** issue (documentation-only pull requests do not require an issue). If this pull request contains code and there isn't yet an issue explaining why you're submitting this pull request, please stop and open a new issue first.
+```
+[ ] Documentation update
+[ ] Bug fix ([template](https://github.com/eslint/eslint/blob/master/templates/bug-report.md))
+[ ] New rule ([template](https://github.com/eslint/eslint/blob/master/templates/rule-proposal.md))
+[ ] Changes an existing rule ([template](https://github.com/eslint/eslint/blob/master/templates/rule-change-proposal.md))
+[ ] Add autofixing to a rule
+[ ] Add a CLI option
+[ ] Add something to the core
+[ ] Other, please explain:
```

Comment:
Yes, this was intentional. Otherwise, people can change it from the issue directly, and that's not what we want here.